docs/source/testing/landing-page.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Setting an Apollo Server landing page
+sidebar_title: Server landing page
+---
+
+> This feature is new in Apollo Server 3.
+
+You can configure the landing page that's served from Apollo Server's base URL. Aside from serving the default landing page, you can serve [GraphQL Playground](#graphql-playground) (an open-source GraphQL IDE), a completely [custom landing page](#custom-landing-page), or you can [disable the landing page entirely](#disabling-the-landing-page).
+
+## Default landing page
+
+Apollo Server serves the following landing page by default:
+
+<img class="screenshot" src="../images/as-landing-page.jpg" alt="Apollo Server default landing page" width="350"/>
+
+This landing page provides a link to [Apollo Sandbox](https://studio.apollographql.com/sandbox), enabling you to test out operations against your server (or any other reachable server):
+
+<img class="screenshot" src="../images/sandbox.jpg" alt="Apollo Sandbox"/>
+
+### Configuring the default landing page
+
+You can configure the default landing page by providing one of the following plugins to the `ApolloServer` constructor:
+
+* `ApolloServerPluginLandingPageLocalDefault` (provide this in non-production environments)
+* `ApolloServerPluginLandingPageProductionDefault` (provide this in production if you provide the landing page at all)
+
+```js{1,7-9}:title=index.js
+const { ApolloServerPluginLandingPageLocalDefault } = require('apollo-server-core');
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  plugins: [
+    ApolloServerPluginLandingPageLocalDefault({
+      footer: false
+    })
+  ]
+});
+```
+
+For available configuration options, see the API reference. (FIXME: add link)
```

Comment:
We might want to explicitly call out some of the options here, but that might be more clear once I finish the API reference (in progress, still feeling under the weather). Can happen later (though we do need to remove the term FIXME to make this mergable)

Reply:
Will give another quick pass here when reviewing #5394